development-updates.md

```diff
@@ -76,7 +85,7 @@ With the gained insight into the protocol, the following weeks serve as a deep d
 | [Another Dev](https://github.com/Another-DevX)               | [Update 3](https://hackmd.io/@btcZWytfSNOGdxJyufkirQ/Bk9f7MlvA) |                                                                 |                                                                 |                                                                                            |
 | [Ashely Yan](https://github.com/AshliaYan)                   | [Update 3](https://hackmd.io/@Ashelyyan/Sk-DjQJDR)              | [Update 4](https://hackmd.io/@Ashelyyan/rkfZvzsvR)              | [Update 5](https://hackmd.io/@Ashelyyan/BJ8oPqz_A)              |                                                                                            |
 | [Ashen](https://github.com/y1cunhui)                         |                                                                 |                                                                 |                                                                 |                                                                                            |
-| [Bastin](https://github.com/Inspector-Butters)               | [Update 3](https://hackmd.io/@Bastin/By8UVwlPA)                 |                                                                 |                                                                 |                                                                                            |
+| [Bastin](https://github.com/Inspector-Butters)               | [Update 3](https://hackmd.io/@Bastin/By8UVwlPA)                 |                                                                 | [Update 5](https://hackmd.io/@Bastin/HyqHfO9OR)                 | [Light Client Support in Prysm](https://hackmd.io/@Bastin/BkdVcRQvA)                       |                                                                                         
```

Comment:
The project should be within the repo `projects/light-client-support-in-prysm.md`